.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
+
+*.log
+*.swp
+
+node_modules
```

Comment:
This is per @aturkewi idea about ignoring all the module noise from students running `npm install`